src/Microsoft.Diagnostics.Monitoring.WebApi/Models/CallStackResults.cs

```diff
@@ -17,9 +17,17 @@ public class CallStackFrame
         [JsonPropertyName("moduleName")]
         public string ModuleName { get; set; }
 
+        [JsonIgnore]
+        internal IList<string> TypeArgs { get; set; } = new List<string>();
```

Comment:
Let's make these:
- `SimpleGenericArgTypes`
- `FullGenericArgTypes`
- `SimpleParameterTypes`
- `FullParameterTypes`
to make clear the `Simple` vs `Full` symmetry and the fact that these are all types.